### PR TITLE
Added PokeAPI data source

### PR DESCRIPTION
**Pull request by marvin-hinkley-vortx**
Gotta catch them all! All the connectors . . .

### CODE REVIEW

**Comment by marvin-hinkley-vortx** [comment]
How long should we generally expect to wait for someone to review our PRs here? I'm some past PRs being taken care of relatively quickly.

File: sources/pokeapi.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "POKEAPI",
+  "name": "PokeAPI",
+  "categories": ["TOOLS", "PUBLIC_DATA"],
+  "organization": "PokeAPI",
```
Comment (pfrisella) [inline sources/pokeapi.json]: there should be a corresponding organizations entry for this

**Comment by pfrisella** [comment]
if everything looks good it can be pretty quick. I suggest you read the instructions and follow other PRs as examples. The org entry is missing.

File: sources/pokeapi.json (hunk 1 above)
Comment (marvin-hinkley-vortx) [reply sources/pokeapi.json]: updated

File: organizations/pokeapi.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "id": "PokeAPI",
```
Comment (pfrisella) [inline organizations/pokeapi.json]:
```suggestion
    "id": "POKEAPI",
```

File: sources/pokeapi.json (hunk 1 above)
Comment (pfrisella) [inline sources/pokeapi.json]:
```suggestion
  "organization": "POKEAPI",
```

**Comment by marvin-hinkley-vortx** [comment]
@pfrisella Did the CLA check fail because I committed your suggested changes and it picked you up as an author? Everything else seems to be in order.

File: organizations/pokeapi.json (hunk 2 above)
Comment (marvin-hinkley-vortx) [reply organizations/pokeapi.json]: updated

File: sources/pokeapi.json (hunk 1 above)
Comment (marvin-hinkley-vortx) [reply sources/pokeapi.json]: updated

**Comment by marvin-hinkley-vortx** [comment]
@pfrisella Do you need anything else from me for this?

**Pull request merged**